src/service/stables.ts

```diff
@@ -67,12 +79,16 @@ export default async function stables(): Promise<TokenModel[]> {
     let units = tokenData.units.value
     let value = prices.value[tokenData.iso4217] * units
 
+    // This adjusts the total supply to account for the reserve owned CUSD that have already been minted
     if (tokenData.symbol === StableToken.cUSD) {
       value -= curveCUSDAmount * prices.value[tokenData.iso4217]
       units -= curveCUSDAmount
 
       value -= multisigCUSDAmount * prices.value[tokenData.iso4217]
       units -= multisigCUSDAmount
+
+      value -= partialReserveCUSDAmount * prices.value[tokenData.iso4217]
+      units -= partialReserveCUSDAmount
```

Comment:
Nor this – basically the partial reserve is equivalent to the normal reserve contract, it will never hold cUSD.